metrix-mapping/src/main/java/com/powsybl/metrix/mapping/TimeSeriesMapperChecker.java

```diff
@@ -431,9 +433,9 @@ private double correctMappedPowerHvdcLine(int point, HvdcLine hvdcLine, MappedPo
         setpoint = applyToleranceThresholdOnTargetP(isMappedSetpoint, setpoint);
 
         if (hvdcLine.getMaxP() < 0) {
-            throw new AssertionError("Equipment '" + id + "' : invalid active limit maxP " + hvdcLine.getMaxP() + " at point " + point);
+            throw new AssertionError(String.format("Equipment '%s' : invalid active limits maxP %s at point %s", id, hvdcLine.getMaxP(), point));
```

Comment:
I think we can keep `limit` in the singular here.

Reply:
corrected